Week3/week3HomeWork/Exercise2/dogYears.js

```diff
@@ -0,0 +1,9 @@
+function calculateDogAge(age) {
+    let puppyAge = age * 7;
```

Comment:
Do you think it's better to use `let` or `const` here?

Reply:
const is better.